src/client/apiclient.ts

```diff
@@ -1,9 +1,12 @@
+import {AxiosRequestConfig, AxiosResponse} from "axios";
```

Comment:
Standard seems to be adding a space after { and before the } braces. You did this on line 7 so suggest doing here and on line 9.

Reply:
I'd suggest leaving it for now as we should really change the everywhere. Maybe a tech debt story?

Reply:
Consensus: do this later as a tidy up task we think. @markpit to create a tech task/story.

Reply:
LFA-1405 created to cover the imports